CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+- Include compiler wrapper arguments (i.e., `CC=compiler arg1 arg2`) in the classification of compilers' tool family. This fixes cases like `clang --driver-mode=cl` being incorrectly detected as needing `clang`-like arguments, rather than `cl`-like arguments. [#????](https://github.com/rust-lang/cc-rs/????).
+
```

Comment:
```suggestion
```

This is automatically generated/overwritten in release PR, created by release-plz

Reply:
Removed with a force-push; interdiff link: https://github.com/rust-lang/cc-rs/compare/ffe0e4164b4d6ac31c62aed7888651d3d15f0c4f..17c805a543f09e60f8c25296f322c9dd12c319bd